.github/workflows/ci.yaml

```diff
@@ -39,8 +39,14 @@ jobs:
         sudo apt install \
           build-essential cmake git ninja-build pkg-config \
           libssl-dev libgeoip-dev zlib1g-dev \
-          libboost-dev libboost-chrono-dev libboost-random-dev libboost-system-dev \
-          libqt5svg5-dev qtbase5-dev qttools5-dev
+          libboost-dev libboost-chrono-dev libboost-random-dev libboost-system-dev
+      # sudo apt install libqt5svg5-dev qtbase5-dev qttools5-dev # the Qt version in the standard repositories is too old...
+
+    # this will be installed under /opt/qt514. CMake will still find it automatically without additional hints
+    - name: install Qt 5.14.2 from an external PPA
+      run: |
+        sudo add-apt-repository ppa:beineri/opt-qt-5.14.2-focal
+        sudo apt install qt514-meta-full
```

Comment:
Doesn't it have separate packages per Qt modules?

Reply:
@glassez Yeah, I could try to minimize the installation footprint, hang on.

Reply:
@glassez fixed.